recipes/intel_mpi.rb

```diff
@@ -36,6 +36,9 @@
     tar -xf l_mpi_#{node['cfncluster']['intelmpi']['version']}.tgz
     cd l_mpi_#{node['cfncluster']['intelmpi']['version']}/
     ./install.sh -s silent.cfg --accept_eula
+    mv rpm/EULA.txt /opt/intel/impi
```

Comment:
should this path also include the intel MPI version like it does below?

Reply:
Yep - nice catch